tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -785,7 +787,10 @@ When('the user browses to the folder {string} on the files page', (folderName) =
     .navigateAndWaitTillLoaded(targetFolder)
 })
 When('the user copies the permalink of the file/folder/resource {string} using the webUI', async function (file) {
-  await client.page.FilesPageElement.filesList().openSharingDialog(file, 'links')
```

Comment:
why changing that? there is still a `openSharingDialog()` in the files list

Reply:
separate function is made to open links dialog! :jack_o_lantern: